tests/test_accessors.py

```diff
@@ -1,8 +1,9 @@
 import numpy as np
+import pytest
 
 import spherely
 
-import pytest
+earth_radius_meters = 6_371_010
```

Comment:
This is not actually used?

Reply:
Ah, good catch @jorisvandenbossche 

Thanks, I've removed it.